include/biosHandler.hpp

```diff
@@ -0,0 +1,65 @@
+#pragma once
+#include <sdbusplus/asio/connection.hpp>
+#include <sdbusplus/bus.hpp>
+
+#include <iostream>
+
+namespace vpd
+{
+/**
+ * @brief A class to operate upon and back up some of the BIOS attributes.
+ *
+ * The class initiates call backs to listen to PLDM service and changes in some
+ * of the BIOS attributes.
+ * On a factory reset, BIOS attributes are initialized by PLDM, this class waits
+ * until PLDM has grabbed a bus name before attempting any syncs.
+ *
+ * It also initiates API to back up back up those data in VPD keywords.
```

Comment:
back up is duplicated.

as "VPD keywords" is not a storage place, is this statement is correct ?

Reply:
> as "VPD keywords" is not a storage place

How did you reach to this assertion?

Reply:
VPD is representing set of data, not the storage itself right.